common/errors/errors.go

```diff
@@ -0,0 +1,20 @@
+package errors
+
+type Error struct {
```

Comment:
A little confusing with similarity to standard errors package, do we want to call it ScootError or something else?

Reply:
Yeah, I actually had it as ScootError at one point, but changed it bc I felt like that name 
1. Didn't tell my anything about why it's different from a standard error
&
2. Implied that the this type of error should in some way be unique to Scoot, when in reality it's just an error with an exit code 

I do agree it's confusing though, maybe CodeError?